example_school_library_decorator/main.rb

```diff
@@ -13,66 +13,70 @@ def initialize
   end
 
   # *List all books
-  def list_books
-    @books.each do |book|
-      puts "Title: \"#{book.title}\", Author: #{book.author}"
+  def list_of_books
+    if @books.length.positive?
+      @books.each { |book| puts "title: #{book.title}, Author: #{book.author}" }
+    else
+      puts 'There are no books created so far'
     end
   end
 
   # *List all people
-  def list_people
-    @people.each do |person|
-      puts "[#{person.classroom}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+  def list_of_people
+    if @people.length.positive?
+      @people.each do |person|
+        puts "Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    else
+      puts 'There are no people created'
```

Comment:
- [ ] The `App` class should not be present in the main.rb file. The main.rb file should serve just one purpose and that is to ask the users for their choice of options in order to start the application. All logic should be kept out of the main.rb file.